lff/convert.py

```diff
@@ -114,6 +114,9 @@ def split_oneliner(match):
             font_author = metadata.get('author', metadata.get('creator', 'converted'))
             print('author = {}'.format(font_author))
             print('license = {}'.format(metadata.get('license', 'unknown')))
+            print('letter_spacing = {:g}'.format(float(metadata.get('LetterSpacing', 3))))
+            print('word_spacing = {:g}'.format(float(metadata.get('WordSpacing', 6.75))))
+            print('line_spacing = {:g}'.format(float(metadata.get('LineSpacingFactor', 1)) * 15.0))
```

Comment:
Did we discuss those keys already?

Reply:
Not yet, but any feedback is welcome :wink:  https://github.com/fontobene/fontobene/pull/29